### ISSUE TITLE
The generate image was not clear

### ISSUE THREAD

**Issue by zwy1996** [SenseTime-FVG/OpenDWM#14]
Hello，thanks for your excellent job. when i use the instruction "python examples/ctsd_generation_example.py -c examples/ctsd_21_6views_image_generation.json -o output/ctsd_21_6views_image_generation"，I got that the generate image was not clear, i used the SD2.1 model. Would you please give me some suggestions？

**Comment by wzhgba** [comment]
1. Is it possible for you to upload the generated image, so I can check if it's about incorrect config, pipeline bug, or limitation of the model?
2. Our model is mainly trained for the multi-view generation.
    1. To generate single view image without training, you can try to edit the cross-view attention mask to identity matrix (the generation quality on multi-view datasets is not guaranteed), and remove all the unexpected view prompts and conditions in the sensor list.
    2. Fine-tune the model for single-view. In this case, you can edit the training config by deleting `enable_crossview`, `enable_rowwise_crossview`, `crossview_block_layers` (SD 3.x only), `crossview_mask` (in the dataset), keeping only 1 expected item in the dataset `sensor_channels`.

**Comment by zwy1996** [comment]
![Image](https://github.com/user-attachments/assets/5740b8d9-0180-4725-bd56-76f0d3a63ebd)，this is my generated image。maybe beacause the output size of model was [256,448]？

**Comment by wzhgba** [comment]
It seems to be the limitation of the model. How about the result of CTSD 3.5?
If the main target is to generate images rather than videos, deleting temporal modules and fine-tuning with higher resolution is also an option.

**Comment by zwy1996** [comment]
sorry，my gpu is 3090, only 24G, when i ran the result of CTSD 3.5, i got oom error. Thanks for your suggestions.
